src/bundle/Resources/views/themes/admin/system_info/my_ibexa.html.twig

```diff
@@ -0,0 +1,70 @@
+{% trans_default_domain "systeminfo" %}
+
+<!-- Tab name: {{ 'tab.name.ibexa'|trans|desc('My Ibexa') }} -->
+
+<section class="ez-fieldgroup">
+    <h2 class="ez-fieldgroup__name">{{ 'ibexa.product'|trans|desc('Product') }}</h2>
+    <div class="ez-fieldgroup__content">
+        <table class="table ez-table ez-table--list ez-table--to-left">
+            <tbody>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.name'|trans|desc('Name') }}</td>
+                    <td class="ez-table__cell">{{ info.name }}</td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.version'|trans|desc('Version') }}</td>
+                    <td class="ez-table__cell">{{ info.release }}</td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.is_dxp'|trans|desc('Is DXP') }}</td>
+                    <td class="ez-table__cell">
+                        <svg class="ez-icon ez-icon--small">
+                            <use xlink:href="{{ ez_icon_path((info.isEnterprise == true) ? 'checkmark' : 'discard') }}"></use>
```

Comment:
wouldn't `ez_icon_path(info.isEnterprise ? 'checkmark' : 'discard')` be enough?

(if so, there are two more occurences below)